### PR TITLE
Add free disk space check to health-checker.py.

### PR DESCRIPTION
**Pull request by ro-tex**
Add free disk space check to health-checker.py.
Move load-average check to health-checker.py.

### CODE REVIEW

File: setup-scripts/health-checker.py [hunk 1]
```diff
@@ -57,6 +66,48 @@ async def run_checks():
...
+        await send_msg(client, "High system load detected: `uptime: {}`".format(uptime_string), force_notify=True)
+
+
+# check_disk checks the amount of free space on the /home partition and issues
+# a warning message if it's under FREE_DISK_SPACE_THRESHOLD GB.
+async def check_disk():
+    # We check free disk space in 1024 byte units, so it's easy to convert.
+    df = os.popen("df --block-size=1024").read().strip()
+    volumes = {}
+    for line in df.split("\n")[1:]:
```
Comment (m-cat) [inline setup-scripts/health-checker.py:88]:
```suggestion
    for line in df.splitlines()[1:]:
```

File: setup-scripts/health-checker.py [hunk 2]
```diff
@@ -26,6 +27,12 @@
 # a lower limit in order to leave some space for additional message text.
 DISCORD_MAX_MESSAGE_LENGTH = 1900
 
+GB = 1 << 20 # converts from KiB to GiB
```
Comment (m-cat) [inline setup-scripts/health-checker.py:30]: I found this line confusing, particularly why does a variable called `GB` convert from KiB to GiB. It made sense after I read `check_disk` but I would rename this variable and/or clarify the comment. Probably just the comment so the following line `50 * GB` continues to make sense.

**Review by m-cat — APPROVED** [review]
Minor non-blockers

File: setup-scripts/health-checker.py (hunk 2 above)
Comment (ro-tex) [reply setup-scripts/health-checker.py:30]: The Python scripts are turning out to be problematic, so we're most probably going to rewrite them in something easier to maintain such as TypeScript. So, for the moment I'll merge this as it and I'll leave some local notes to improve the documentation.

**Pull request merged**